### PR TITLE
xplot working in jupyter and nteract

### PR DESCRIPTION
**Pull request by colombod**
Handle plotly loading via require js  in both jupyter and nteract

### CODE REVIEW

File: XPlot.DotNet.Interactive.KernelExtensions/XPlotKernelExtension.cs [hunk 1]
```diff
@@ -30,24 +31,37 @@ public string GetChartHtml(PlotlyChart chart)
...
+    script.setAttribute(""src"", ""https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js""); 
+    script.onload = function(){
+        renderPlotly();
+    };
+    document.getElementsByTagName(""head"")[0].appendChild(script); 
+}
+else {
+    renderPlotly();
+}");
             newScript.AppendLine("</script>");
```
Comment (jonsequitur) [inline XPlot.DotNet.Interactive.KernelExtensions/XPlotKernelExtension.cs]: Why not use `PocketView` rather than add a dependency on HtmlAgilityPack?

Comment (colombod) [reply XPlot.DotNet.Interactive.KernelExtensions/XPlotKernelExtension.cs]: we need parsing anyway to select the div and script element from the xplot html

Comment (jonsequitur) [reply XPlot.DotNet.Interactive.KernelExtensions/XPlotKernelExtension.cs]: Can't they just be rendered as separate script elements?

Comment (colombod) [reply XPlot.DotNet.Interactive.KernelExtensions/XPlotKernelExtension.cs]: The script tag injects the other script element conditionally if requrirejs is not defined, they cannot be 2 tags

**Pull request merged**